elixir/lib/mix/tasks/generate.ex

```diff
@@ -359,6 +379,53 @@ defmodule Mix.Tasks.Contracts.Generate do
     "object" not in types_list
   end
 
+  @doc """
+    Generates the cloud events conversion functions for the contract
+  """
+  def trento_cloudevent_conversion_functions() do
+    quote do
+      @doc """
+        Serialize and return a module struct from a cloud event json
+      """
+      def serialize_from_cloud_event(json_cloud_event) do
```

Comment:
Could we maybe add a signature to the method?
In this case it needs a binary, right?
Something like:
```
@spec serialize_from_cloud_event(json :: binary()) ::
          {:ok, t()} | {:error, any}
```